test_regress/t/t_lint_misindent_bad.pl

```diff
@@ -12,7 +12,7 @@
 
 lint(
     fails => 1,
-    verilator_flags2 => ["--lint-only -Wall -Wno-DECLFILENAME"],
+    verilator_flags2 => ["--lint-only -Wall -Wno-DECLFILENAME -Wno-UNUSEDLOOP"],
```

Comment:
Please change the test to not need to disable the warning, or add a verilator lint_off inside it.